announcement/announcement-api/api/src/bundle/announcement_ca.properties

```diff
@@ -25,6 +25,7 @@ java.alert.youacc.pes=No teniu autoritzaci\u00f3 per accedir al(s)  missatge(s)
 java.alert.thisis=Un altre usuari est\u00e0 editant aquest element. Proveu-ho m\u00e9s tard.
 ### (moot) java.alert.youcreate = you don't have permissions to create this announcement -
 java.alert.youchoosegroup=Heu de seleccionar almenys un grup.
+java.alert.youchooserole=Heu de seleccionar almenys un rol.
```

Comment:
Put all the new strings together at the end of the file, put a comment for S2U-36.